lib/plugins/zero-conf/blockcypher/blockcypher.js

```diff
@@ -21,17 +24,28 @@ function authorize (account, toAddress, cryptoAtoms, cryptoCode) {
       })
 
       const confidence = account.confidenceFactor
-
+      const isRBFEnabled = account.rbf
```

Comment:
Shouldn't this have a UI change as well? How did you add this rbf option?

Reply:
There's a card attached with the UI part